### PR TITLE
Update automatically-generated-release-notes.md

### LINKED ISSUE #18155: Clarify YAML config for automated release notes
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes)

### What part(s) of the article would you like to see updated?

Examples of the YAML configuration when trying to exclude labels from categories.  My use case is that we don't label PRs, but dependabot does.  When we generate release notes, I'd really love to see our feature updates before the dependabot labels, but it took me a long time to figure out how to configure the exclude key in the YAML to display my non-labeled Features first.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by katelovescode**
clarifies exclusion configuration example

### Why:

Closes [18155](https://github.com/github/docs/issues/18155)

### What's being changed:

Adds a description to the existing example and adds an additional example that shows the exclusion method explicitly:

===============
A configuration for a repo that doesn't tag Pull Requests but wants to separate out Dependabot automated PRs in release notes (note: `labels: '*'` is required to display a catchall category)

```yaml{:copy}
# .github/release.yml
changelog:
  categories:
    - title: 🏕 Features
      labels:
        - '*'
      exclude:
        labels:
          - dependencies
    - title: 👒 Dependencies
      labels:
        - dependencies
```

================

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@katelovescode Thanks so much for opening a PR! I'll get this triaged for review 💖

**Comment by cmwilson21** [comment]
We appreciate your patience as we are working through our backlog 💛

File: content/repositories/releasing-projects-on-github/automatically-generated-release-notes.md [hunk 1]
```diff
@@ -102,6 +104,26 @@ changelog:
 ```
 {% endraw %}
 
+A configuration for a repo that doesn't tag Pull Requests but wants to separate out Dependabot automated PRs in release notes (note: `labels: '*'` is required to display a catchall category)
```
Comment (mchammer01) [inline content/repositories/releasing-projects-on-github/automatically-generated-release-notes.md:107]: we avoid using `repo` and `PR` in the docs, so suggesting this:
```suggestion
A configuration for a repository that doesn't tag pull requests but where we want to separate out Dependabot automated pull requests in release notes (`labels: '*'` is required to display a catchall category)
```

**Review by mchammer01 — COMMENTED** [review]
@katelovescode - thanks for your patience with us ❤️ 
Your PR looks good ✨ 
I've made suggestions to keep the updates in line with our content model. I'll submit them and will merge your PR ⚡

File: content/repositories/releasing-projects-on-github/automatically-generated-release-notes.md [hunk 2]
```diff
@@ -102,6 +104,26 @@ changelog:
 ```
 {% endraw %}
 
+A configuration for a repository that doesn't tag pull requests but where we want to separate out Dependabot automated pull requests in release notes (`labels: '*'` is required to display a catchall category)
```
Comment (mchammer01) [inline content/repositories/releasing-projects-on-github/automatically-generated-release-notes.md:107]: Use product variable
```suggestion
A configuration for a repository that doesn't tag pull requests but where we want to separate out {% data variables.product.prodname_dependabot %} automated pull requests in release notes (`labels: '*'` is required to display a catchall category)
```

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Comment by tharkhay4** [comment]
> clarifies exclusion configuration example
> 
> ### Why:
> Closes [18155](https://github.com/github/docs/issues/18155)
> 
> ### What's being changed:
> Adds a description to the existing example and adds an additional example that shows the exclusion method explicitly:
> 
> =============== A configuration for a repo that doesn't tag Pull Requests but wants to separate out Dependabot automated PRs in release notes (note: `labels: '*'` is required to display a catchall category)
> 
> ```
> # .github/release.yml
> changelog:
>   categories:
>     - title: 🏕 Features
>       labels:
>         - '*'
>       exclude:
>         labels:
>           - dependencies
>     - title: 👒 Dependencies
>       labels:
>         - dependencies
> ```
> 
> ================
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")
https://github.com/github/docs/pull/18156#issue-1245811159

**Comment by tharkhay4** [comment]
# .github/release.yml
changelog:
  categories:
    - title: 🏕 Features
      labels:
        - '*'
      exclude:
        labels:
          - dependencies
    - title: 👒 Dependencies
      labels:
        - dependencies

**Comment by tharkhay4** [comment]
Config file not present. Please add config file at .github/engineering-scorecard.yaml in repository: docs

**Pull request merged**